bambi/utils.py

```diff
@@ -19,3 +24,55 @@ def multilinify(sequence, sep=","):
     """Make a multi-line string out of a sequence of strings."""
     sep += "\n"
     return "\n" + sep.join(sequence)
+
+
+def c(*args):  # pylint: disable=invalid-name
```

Comment:
I would argue for a longer more descriptive name

Reply:
I know that `c()` may not be the most self-describing name, but it's a convention I borrowed from the R language where `c()` is used a lot. On the other side, I'm 100% open to considering other alternatives and seeing if they work.

Reply:
@aloctavodia do you have any opinion about it?